src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  ElementRef,
+  Injectable,
+  Renderer2,
+  RendererFactory2
+} from '@angular/core';
+
+import {
+  SkyMediaBreakpoints
+} from '../media-query';
+
+/* tslint:disable */
+const SKY_TABBABLE_SELECTOR = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled]):not([tabindex=\'-1\'])',
+  'button:not([disabled]):not([tabindex=\'-1\'])',
+  'select:not([disabled]):not([tabindex=\'-1\'])',
+  'textarea:not([disabled]):not([tabindex=\'-1\'])',
+  'iframe',
+  'object',
+  'embed',
+  '*[tabindex]:not([tabindex=\'-1\'])',
+  '*[contenteditable=true]'
+].join(', ');
+/* tslint:enable */
+
+@Injectable()
+export class SkyAdapterService {
+  private renderer: Renderer2;
```

Comment:
Could we place an empty line above this? Personal preference :)

Reply:
Done.